src/helpers/featureSwitch.ts

```diff
@@ -0,0 +1,19 @@
+type StringMap = { [key: string]: any };
+
+export const getFeatureFlags = (environmentVariables: StringMap) => {
+    return Object.fromEntries(
+        Object.entries(environmentVariables)
+        .filter(variableEntry => variableEntry[0].startsWith("REACT_APP_FEATURE"))
```

Comment:
[envVarName, envVarValue] => envVarName.startsWith("REACT_APP_FEATURE"))

Reply:
Or something similar to make it more readable?

Reply:
@BastinJafari, updated